src/main/java/com/enderio/base/common/item/darksteel/upgrades/EmpoweredUpgrade.java

```diff
@@ -22,37 +23,38 @@ public class EmpoweredUpgrade extends TieredUpgrade<EmpoweredUpgradeTier> {
 
     private final ForgeConfigSpec.ConfigValue<Integer> powerUsePerDamagePoint = BaseConfig.COMMON.DARK_STEEL.EMPOWERED_ENERGY_PER_DAMAGE;
 
-    private EnergyStorage storage;
+    @Nullable private EnergyStorage storage;
```

Comment:
nullable on line before the field declaration

Reply:
This file is not a part of my PR. I have not touched it at all. The field is nullable on my side. Probably a residue of so many Merges

Reply:
Nevermind. I don't remember editing that file ever but it does have changes from my side (its a change from 7 months ago lol ). I will make the required changes